actionview/lib/action_view/layouts.rb

```diff
@@ -255,7 +255,7 @@ def _conditional_layout?
       # true::   raise an ArgumentError
       # nil::    Force default layout behavior with inheritance
       #
-      # Return value of Proc & Symbol arguments should be String, false, true or nil
+      # Return value of +Proc & Symbol+ arguments should be +String+, +false+, +true+ or +nil+
```

Comment:
I think +Proc & Symbol+ should be separate, and perhaps the ampersand should be spelled out ("and").